### DISCUSSION TITLE
context menu

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7292] [Help]
how can we create context menu

**Comment by melohagan** [comment]
Speaking of context, could I have a little more context please?

**Reply by DevrayePatil** [reply]
![image](https://user-images.githubusercontent.com/54846160/185552253-17c58a34-0f83-4a52-8c1d-249a3ee954a1.png)
as like this

**Comment by melohagan** [comment]
Hey @DevrayePatil 

It's not great, but I was able to come up with this using an embed component and an options picker:

1. Add an options picker with custom options: Approve, Reject.
2. Add an embed component to your screen with the following style
```html
<style>
button.spectrum-Picker {
  width: 25% !important;
  background-color: transparent !important;
  border: none !important;
}
.spectrum-Icon {
  display: none !important;
}
.spectrum-Picker-label.is-placeholder {
  visibility: hidden;
  position: relative;
}
.spectrum-Picker-label.is-placeholder:after {
  visibility: visible;
  position: absolute;
  top: 0; 
  left: 0;
  content: "⋮";
  font-size: x-large;
  font-style: normal;
  font-weight: bold;
}
</style>
```

**Result**
![contextmenu](https://user-images.githubusercontent.com/101575380/185587311-844b159f-f722-4ec0-9131-23549df9701b.gif)
